pkg/configure/oneagent/pmc/pmc.go

```diff
@@ -0,0 +1,42 @@
+package pmc
+
+import (
+	"os"
+	"path/filepath"
+
+	"github.com/Dynatrace/dynatrace-bootstrapper/pkg/configure/oneagent/pmc/ruxit"
+	"github.com/go-logr/logr"
+	"github.com/spf13/afero"
+)
+
+const (
+	InputFileName = "ruxitagentproc.json"
```

Comment:
Does this need to be exported?

Reply:
right now, no.

but I plan to (try to) use this as a library.
In every package, that relays on the `--input-directory`, there is 1 or multiple exported `InputFileName`.
So when we want to create the `Secret` that will be mounted to `--input-directory`, we don't have to go in blind.